fml/message_loop_task_queues.cc

```diff
@@ -125,6 +125,7 @@ size_t MessageLoopTaskQueues::GetNumPendingTasks(TaskQueueId queue_id) {
 void MessageLoopTaskQueues::AddTaskObserver(TaskQueueId queue_id,
                                             intptr_t key,
                                             fml::closure callback) {
+  FML_CHECK(callback != nullptr) << "Observer callback must be non-null.";
```

Comment:
This can remain a DCHECK. We have already made sure a null check happens in release.